framgia/sph-ut-uni-charts
# Create webhook and update Backlog Issue

# How to Test
- You should host your localhost to internet first bye running this command:
```
ssh -R 80:localhost:13000 localhost.run
```
- Then get the link generated by the tunneling service, similar to the image below:
<img width="844" alt="Screen Shot 2022-04-29 at 4 14 02 PM" src="https://user-images.githubusercontent.com/13476073/165908031-a8828dda-b578-44c0-848d-e8b246bbe1f0.png">

- Copy the link and paste it to your clockify webhook page, to locate where just follow this docs https://clockify.me/developers-api
- After that try to record the time with the description format `<BACKLOG ISSUE KEY> Task Title` 

Example:
```
UT_UNICHARTS-1 Sample 1
```

- Then try to stop the timer and check backlog issue if it is updated
